### PR TITLE
Issue #217 Readme update

### PR DESCRIPTION
**Pull request by kksingh711**
updated readme with installation and running steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,19 @@ Turn this into trading application
...
 - If you need any other help, feel free to ping on crypto channel at discord.
 
+**Installation**
+* `git clone<repository-url>` this repository
+* `cd website-crypto`
+* `npm install`
+
+**Runing/Development**
+* `npm run build`
+* `npm run start`
```
Comment (swarajpure) [inline README.md:29]: `build` is for production build
`start` is to serve the the production build, made by `build`

We use `dev` for development

Can you please make these changes?

Thank you for the pull request! 🥳

Comment (kksingh711) [reply README.md]: Please review it once again I made the changes.

Comment (swarajpure) [reply README.md]: Please read that again.... We use `dev` for development purposes

Comment (kksingh711) [reply README.md]: Sorry for the inconvenience. I think the required changes are made please review.

Comment (swarajpure) [reply README.md]: Almost there. Shouldn't have removed `build` and `start`

You can mention these two commands below the dev one. Saying
To build: npm run build
To serve the production build: npm run start

Make sure these are AFTER the dev command, so users are not confused. Thank you so much!

I'd like to invite you to our Discord Server 😉
We have such good projects which are in React, Ember, Vue for frontend and Node, Express in backend. We keep having technical and non-technical discussions there. Here's the link to join:  https://discord.gg/CjwSPnjB

Comment (kksingh711) [reply README.md]: Thank you for your guidance. The said changes are made please review.

**Review by swarajpure — APPROVED** [review]
Looks good to me!

Thanks! Would like to see you in the server 😊

**Pull request merged**